services/microbadger/microbadger-size.service.js

```diff
@@ -4,6 +4,29 @@ const prettyBytes = require('pretty-bytes')
 const BaseMicrobadgerService = require('./microbadger-base')
 const { NotFound } = require('..')
 
+const documentation = `
+<p>
+  The MicroBadger API can sometimes be a bit temparamental when it comes to retrieving the size of your image. 
```

Comment:
I still think it's `..era..` if I'm not mistaken

Reply:
Sorry, I'm struggling with this one tonight... 😕

Reply:
😄 no worires